eland/ml/__init__.py

```diff
@@ -15,9 +15,6 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
-from eland.ml.ml_model import ImportedMLModel, MLModel
```

Comment:
Continue to import the name, but you can remove the name from `__all__` as you're doing now.

Reply:
So, this will throw a lint error
eland/ml\__init__.py:18:1: F401 'eland.ml.ml_model.ImportedMLModel' imported but unused

So we must include this in `__all__`